docs/src/testutils.md

```diff
@@ -1,9 +1,16 @@
 # TestUtils
 
-Provides test fakes, [`FakeTemplate`](@ref) and [`FakeModel`](@ref), that are useful for
-testing downstream dependencies, and [`test_interface`](@ref) for testing the model's API
-has been correctly implemented.
+```@docs
+TestUtils
+```
+
+## Interface Tests
+```@docs
+test_interface
+```
 
+## Test Fakes
 ```@autodocs
 Modules = [Models.TestUtils]
+Filter = t -> occursin("Fake", string(t))
```

Comment:
This is really cool.
REALLY COOL

Reply:
😆 It's kinda nice that is filter through the actual types (although obviously here we stringify them and filter on strings), but if e.g. we wanted a Templates-only section we could `t -> t <: Template` without worrying about if `Template` was in the name.